evalai/challenges.py

```diff
@@ -21,12 +22,19 @@ def challenges(ctx):
 
 @click.group(invoke_without_command=True, name='list')
 @click.pass_context
-def list_challenges(ctx):
+@click.option('-participate', default='false',
```

Comment:
Optional args which are not single character should be --, i.e. `--participate`. Also, I would mark this as `--participant` instead of `--participate`.

Also set the default to `False` so that the type if inferred as Boolean.

Reply:
That looks much better! :D